src/scicookie/{{cookiecutter.project_slug}}/{{cookiecutter.package_slug}}/__init__.py

```diff
@@ -20,6 +23,7 @@ def get_version() -> str:
 
 version: str = get_version()
 
+__version__: str = version
+{%- endif %}
 __author__ = {{ QUOTE }}{{ cookiecutter.author_full_name }}{{ QUOTE }}
 __email__ = {{ QUOTE }}{{ cookiecutter.author_email }}{{ QUOTE }}
```

Comment:
Also, its weird that there is a typing annotation for `__version__` but not for other dunder variables.